dmtx.h

```diff
@@ -24,6 +24,8 @@ extern "C" {
 
 /* Time headers required for DmtxTime struct below */
 #include <time.h>
+#include <stdbool.h>
```

Comment:
Well, this will break the builds for people who still patches buildsystem to use ANSI C instead C99, but it's okay, as buildsystem was migrated about 6 years ago already, and even C99 is 20 yrs old already.

Reply:
no problem, I'll change bool with DmtxBoolean ;)